source/element-kinds.md

```diff
@@ -58,11 +61,12 @@ Example:
 ```{code} yaml
 bb1:
   kind: BeamBeam
-  sigma_x: 0.1e-3
-  sigma_y: 0.1e-3
-  sigma_z: 5.0e-2
-  energy: 1.0e10
-  N_particle: 1.0e11
+  beambeamP:
```

Comment:
I think this should be `BeamBeamP` (not `beambeamP`), no? At least I see `BeamBeamP` above.

Reply:
@EZoni Thanks and fixed.